### PR TITLE
l10n: Reconstruction of sentences and change l10n

### PR DESCRIPTION
**Pull request by Valdnet**
1. Proposed changes of sentences by a translator from Transifex.
2. Changing the l10n directory.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by dartcafe — CHANGES_REQUESTED** [review]
Ah. No. It describes to set the gap between the option and a calendar event in hours. Maybe the comma has to be removed.

**Comment by Valdnet** [comment]
> Maybe the comma has to be removed.

The conjunction should then be used.

File: src/js/components/Settings/UserSettings/CalendarSettings.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@
 				{{ t('polls', 'Use calendar lookup for conflicting calendar events') }}
 			</CheckboxRadioSwitch>
 			<div v-show="calendarPeek" class="settings_details">
-				{{ t('polls', 'Activate all calendars to consult for the search of conflicting events.') }}
+				{{ t('polls', 'Activate all calendars and search for conflicting events.') }}
```
Comment (rakekniven) [inline src/js/components/Settings/UserSettings/CalendarSettings.vue:30]: Change requested:
Activate all calendars to search for conflicting events

**Review by rakekniven — DISMISSED** [review]

File: src/js/components/Settings/UserSettings/CalendarSettings.vue [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 		<div class="user_settings">
 			<p class="settings-description">
-				{{ t('polls', 'Set the time interval in hours, a calendar event must be finished before a date option, to get accounted to the conflict check.') }}
+				{{ t('polls', 'Set the time interval in hours. A calendar event, whose time interval is set in number of hours, must finish before the given date in order to be included in the conflict check.') }}
```
[the remark this answers is not in the document]
Comment (dartcafe) [reply src/js/components/Settings/UserSettings/CalendarSettings.vue:45]: Let's use this: 
'Specify in which period (in hours) before the option existing appointments should be included in the search results.'

File: src/js/components/Settings/UserSettings/CalendarSettings.vue [hunk 3]
```diff
@@ -55,7 +55,7 @@
 
 		<div class="user_settings">
 			<p class="settings-description">
-				{{ t('polls', 'Set the time interval in hours, a calendar event must start after a date option, to get accounted to the conflict check.') }}
+				{{ t('polls', 'Set the time interval in hours. A calendar event, whose time interval is set in number of hours, must start after the given date in order to be included in the conflict check.') }}
```
[the remark this answers is not in the document]
Comment (dartcafe) [reply src/js/components/Settings/UserSettings/CalendarSettings.vue:58]: And: 
'Specify in which period (in hours) before the option existing appointments should be included in the search results.'

**Review by dartcafe — CHANGES_REQUESTED** [review]

**Review by rakekniven — DISMISSED** [review]

File: src/js/components/Settings/UserSettings/CalendarSettings.vue [hunk 4]
```diff
@@ -55,7 +55,7 @@
 
 		<div class="user_settings">
 			<p class="settings-description">
-				{{ t('polls', 'Set the time interval in hours, a calendar event must start after a date option, to get accounted to the conflict check.') }}
+				{{ t('polls', 'Specify in which period (in hours) before the option existing appointments should be included in the search results.') }}
```
Comment (dartcafe) [inline src/js/components/Settings/UserSettings/CalendarSettings.vue:58]: should be `after` here. ;-)

**Review by dartcafe — CHANGES_REQUESTED** [review]

**Review by dartcafe — APPROVED** [review]

**Comment by dartcafe** [comment]
finally. 🎉 😄

**Pull request merged**